develop/smart-contracts/libraries/viem.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Viem for Asset Hub Smart Contracts
```

Comment:
viem should always be lowercase, even in titles and if it's the first word in a sentence